l10n-screenshots.sh

```diff
@@ -25,5 +32,6 @@ for lang in $LOCALES; do
         --derived_data_path l10n-screenshots-dd \
         --erase_simulator --localize_simulator \
         --devices "iPhone 8" --languages "$lang" \
-        --output_directory "l10n-screenshots/$lang" > "l10n-screenshots/$lang/snapshot.log" 2>&1
```

Comment:
@JohanLorenzo removing this partt: `"l10n-screenshots/$lang/snapshot.log"` we may be missing the log file? It has been very helpful to look at it while debugging issues when launching these tests...

Reply:
Sure, you are right, makes sense to have the logs visible in Bitrise somehow. Thanks for the explanation!